src/ontology/cl-edit.owl

```diff
@@ -28247,6 +28248,17 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:16230044"^^xsd:string) r
 AnnotationAssertion(rdfs:label obo:CL_4030025 "renal cortical fibroblast")
 EquivalentClasses(obo:CL_4030025 ObjectIntersectionOf(obo:CL_0000057 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0005270)))
 
+# Class: obo:CL_4032000 (club-like cell of the urethra epithelium)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:30566875"^^xsd:string) Annotation(oboInOwl:hasDbXref "PMID:31390564"^^xsd:string) Annotation(oboInOwl:hasDbXref "PMID:32497356"^^xsd:string) obo:IAO_0000115 obo:CL_4032000 "An epithelial cell of the urethra that is similar to lung club cells in their expression of SCGB1A1. Club-like cells of the urethra epithelium are similar to lung club cells in their enrichment of immunomodulatory programs.")
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4032000 <http://orcid.org/0000-0001-6757-4744>)
+AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_4032000 "club cell of the urethra epithelium")
+AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_4032000 "club-like cell of the prostatic urethra epithelium")
```

Comment:
Is there a dbxref to support this exact synonym?

Reply:
if you mean the "-like" part, nope, but see above for reasoning

Reply:
Perhaps using your ORCiD as a dbxref would be appropriate.

Reply:
do we do that for synonyms? but sure, happy to :)

Reply:
In practice, I try to find a source to justify synonyms I add. 
In this case, @shawntanzk is the source :)